### ISSUE TITLE
The action "Follow the link labeled next or >" does not follow the <input type="button">

### ISSUE THREAD

**Issue by kalokng** [philc/vimium#1935]
Hi,
Currently I am using 1.53, in the following page the popup did not show when I press ]]
```html
<html>
<body>
<input type="button" value="next" onclick="prompt('test')" />
</body>
</html>
```
Is it intended or a bug?
Thanks in advance.

**Comment by smblott-github** [comment]
It looks like we don't "click" buttons currently.

A PR would be welcome.  It should just be a matter of fixing up the XPath in `findAndFollowLink`.

**Comment by smblott-github** [comment]
Fixed in #2117.

**Comment by sco-tt** [comment]
Sorry I wasn't clear in my PR -- I haven't actually fixed this particular issue yet, that has to do with looking for matches in the element value attribute, not innerText.

I just wanted to make a separate commit for similar issues I encountered while debugging this. I plan on getting to this particular issue soon.
